lib/jsdom/living/xhr/FormData-impl.js

```diff
@@ -170,9 +169,9 @@ function constructTheFormDataSet(form, submitter) {
 }
 
 function isBlob(obj) {
-  return obj !== null && typeof obj === "object" && blobSymbols.buffer in obj;
+  return obj instanceof Blob;
```

Comment:
Even using symbols, it means `Object.assign({},new Blob())` would pass. Is perfect encapsulation like the browser does even possible or worth shooting for here, that it's worth sacrificing trying to stick to a public api? There's always one weird trick that bypasses things.